redaxo/src/addons/cronjob/lib/console/run.php

```diff
@@ -0,0 +1,36 @@
+<?php
```

Comment:
diese datei liegt jetzt unter "console". in ydeploy hieß es "command". im core auch "console". wie wollen wir es nennen ;)

Reply:
Ich glaube, ich würde hier `command` bevorzugen, da der Begriff so auch im Klassenname vorkommt. Aber letztlich ist es mir egal. :)